conwayconsole/Program.cs

```diff
@@ -22,7 +22,9 @@ public static async Task Main(params string[] args)
 
       Console.Clear();
 
-      for (IReadableBoard board = initialBoard;; board = game.Turn())
+      Console.CursorVisible = false;
```

Comment:
You need to ensure that the cursor visibility is restored when the program exits